Stripe/UIView+Stripe_SafeAreaBounds.m

```diff
@@ -0,0 +1,38 @@
+//
+//  UIView+Stripe_SafeAreaBounds.m
+//  Stripe
+//
+//  Created by Ben Guo on 12/12/17.
+//  Copyright © 2017 Stripe, Inc. All rights reserved.
+//
+
+#import "UIView+Stripe_SafeAreaBounds.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@implementation UIView (Stripe_SafeAreaBounds)
+
+- (CGRect)boundsWithHorizontalSafeAreaInsets {
+    if (@available(iOS 11.0, *)) {
+        UIView *view = self;
+        UIEdgeInsets insets = UIEdgeInsetsZero;
+        while (view.superview != nil || UIEdgeInsetsEqualToEdgeInsets(insets, UIEdgeInsetsZero)) {
+            view = view.superview;
```

Comment:
This seems like not how `safeAreaInsets` is meant to be used. Is there some other reason the insets aren't being propagated to subviews like described in the docs?

Reply:
Ah whoops, yeah this isn't necessary – I was confused by the docs at first, and thought it didn't propagate.